ui/components/explore_data.py

```diff
@@ -1,4 +1,41 @@
-from typing import List
+import os
+import json
+import logging
+from urllib.parse import quote
+from typing import List, Dict
+logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
 
-def search_relevant_urls(keywords: List[str])-> List[str]:
-    return keywords
\ No newline at end of file
+DATA_PATH = "C:\\Users\\ohyeo\\OneDrive\\rip\\open-data-main\\data\\USA\\"
+
```

Comment:
I think we can use this one;
https://raw.githubusercontent.com/climatepolicyradar/open-data/main/data/USA/